scripts/redis_crash_handler.py

```
@@ -0,0 +1,1134 @@
+    # Step 5: Check for existing issue and crash duplication
+    print("Checking for existing issue...")
+    github = GitHubIssueManager(github_token, issue_repo, project_id)
+    existing_issue, is_same_crash = github.find_or_get_issue(customer.email, args.namespace, crash)
+    
+    if existing_issue and is_same_crash:
+        # Same crash already reported in this issue - add comment to track occurrences
+        print(f"✅ Exact same crash already reported in issue #{existing_issue}")
+        print("   Adding comment to track crash occurrence...")
+        
+        # Add a brief comment to track that this crash happened again
+        duplicate_comment = f"**Same crash detected again at {timestamp}**\n\nPod: {args.pod}, Namespace: {args.namespace}, Cluster: {args.cluster}"
```

Ruff (0.14.14)

[error] Local variable `duplicate_comment` is assigned to but never used

Remove assignment to unused variable `duplicate_comment`

(F841)